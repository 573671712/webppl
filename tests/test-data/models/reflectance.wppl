var model = function() {
  var reflectance = gaussian(1, 1);
  var illuminance = gaussian(3, 0.5);
  var lightness = reflectance * illuminance;
  factor(gaussianERP.score([5, 0.1], lightness))
  return reflectance
};
