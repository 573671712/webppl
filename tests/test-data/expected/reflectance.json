{
  "mean": 1.6,
  "std": 0.4
}
